src/sections/Meshery/Meshery-integrations/Integration.style.js

```diff
@@ -29,6 +29,12 @@ export const HoneycombGrid = styled.div`
     color: #1e2117;
     font-size: 0.875rem;
     cursor: pointer;
+    transition: all .1s ease-in-out;
+
+    &:hover {
+      opacity: 0.8;
+      box-shadow: rgb(60 73 79) 0px 0px 6px -2px ${props => props.theme.DarkTheme ? "#FFFFFF" : "#3c494f"};
```

Comment:
This is incorrct syntax.
Please build and check the code before making a commit, it helps save time of reviewers.

Reply:
Can we expect an update here, @sudhanshutech ?

Reply:
> Can we expect an update here, @sudhanshutech ?

Soon @Nikhil-Ladha , just slipped of my mind because busy on some other issues. I 'll soon update the changes